lib/Controller/Report.php

```diff
@@ -745,6 +745,7 @@ public function savedReportExport($savedreportId, $reportName)
 
             try {
                 $mpdf = new \Mpdf\Mpdf([
+                    'tempDir' => realpath(__DIR__ . '/../../library/temp'),
```

Comment:
You will need to look up the LIBRARY_LOCATION setting here instead, because a relative path won't always work